app/views/shared/_footer.html.erb

```diff
@@ -1,41 +1,130 @@
+  >
     <button class="dm-button--unstyled-primary dm-return-to-top width-auto">Return to top</button>
   </div>
-  <%# footer actions %>
-  <div class="bg-base-lightest">
-    <div class="desktop:padding-y-2">
-      <div class="grid-row desktop:margin-left-4">
-        <a class="desktop:display-inline-block desktop:grid-col-auto desktop:padding-0 desktop:border-top-0 border-top-1px border-base-light desktop:margin-right-8 grid-col-12 padding-y-2 padding-x-2 dm-footer-link" href="/">
-          Home
-        </a>
-        <a href="/about#contact-us" class="desktop:display-inline-block desktop:grid-col-auto desktop:padding-0 desktop:border-top-0 border-top-1px border-base desktop:margin-right-8 grid-col-12 padding-y-2 padding-x-2 dm-footer-link" >
-          Contact us
-        </a>
-        <a class="desktop:display-inline-block desktop:grid-col-auto desktop:padding-0 desktop:border-top-0 border-top-1px border-base desktop:margin-right-8 grid-col-12 padding-y-2 padding-x-2 dm-footer-link" href="<%= nominate_a_practice_path %>" >
-          Nominate an innovation
-        </a>
-        <a class="desktop:display-inline-block desktop:grid-col-auto desktop:padding-0 desktop:border-top-0 border-top-1px border-base desktop:margin-right-8 grid-col-12 padding-y-2 padding-x-2 dm-footer-link" href="/open-calls" >
-          Open calls
-        </a>
-        <a href="https://www.va.gov/privacy-policy" target="_blank" class="desktop:display-inline-block desktop:grid-col-auto desktop:padding-0 desktop:border-top-0 border-top-1px border-base desktop:margin-right-8 grid-col-12 padding-y-2 padding-x-2 dm-footer-link">
-          Privacy policy
-        </a>
-        <a href="<%= terms_and_conditions_path %>" class="desktop:display-inline-block desktop:grid-col-auto desktop:padding-0 desktop:border-top-0 border-top-1px border-base grid-col-12 padding-y-2 padding-x-2 dm-footer-link">
-          Terms and conditions
-        </a>
-      </div>
-    </div>
-  </div>
 
-  <div class="bg-ink padding-y-3 padding-left-2 desktop:padding-left-0">
-    <div class="grid-row desktop:margin-left-4">
-      <img class="display-block maxw-5" src="<%= asset_path( 'va-seal.png' ) %>" alt="Seal of the Department of Veterans Affairs"/>
-      <p class="margin-0 margin-left-1 text-white">
-        <span class="text-heavy font-sans-md">Diffusion Marketplace</span><br>
-        <span class="font-sans-2xs">Department of Veterans Affairs</span>
-      </p>
-    </div>
+  <div class="usa-footer__primary-section bg-primary-darker padding-y-4 desktop:padding-x-7">
+    <nav class="usa-footer__nav margin-x-auto padding-0" aria-label="Footer navigation">
+      <section class="usa-footer__primary-content usa-footer__primary-content--collapsible">
+        <img src="<%= asset_path( 'dm-footer-logo.png' ) %>" alt="VA | Diffusion Marketplace footer logo"/>
```

Comment:
I'd recommend removing `footer logo` from the alt text. Since this is a text logo, the info can be read as "VA Diffusion Marketplace".